test/LetConst/letlet.baseline

```diff
@@ -0,0 +1,13 @@
+*** Running test #1 (0): 'let' should not be an allowed name in let declarations
+PASSED
+*** Running test #2 (1): 'let' should not be an allowed name in const declarations
+PASSED
+*** Running test #3 (2): 'let' should not be an allowed name in destructuring let declarations
+PASSED
+*** Running test #4 (3): 'let' should not be an allowed name in destructuring const declarations
+PASSED
+*** Running test #5 (4): 'let' should not be an allowed name in 'for(let .. in ..)' declarations
+PASSED
+*** Running test #6 (5): 'let' is OK if used in var declarations
+PASSED
+Summary of tests: total executed: 6; passed: 6; failed: 0

```

Comment:
Instead of using a baseline you can use a summary argument to `ch.exe` to choose to print a terse "pass/fail" message that `rl.exe` will recognize in the absence of a baseline.

Take a look at another test that uses `UnitTestFramework.js` but doesn't have a baseline specified in `rlexe.xml`.  Not sure if there are examples under `test/LetConst`, but there are definitely examples under `test/es6`.